### PR TITLE
Feature mail sample

### PR DESCRIPTION
**Pull request by tpryan**
Tried to match general DPE style.

### CODE REVIEW

File: appengine/standard/mail/test/mocks/Message.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace google\appengine\api\mail;
+
+class Message
+{
+
```
Comment (bshaffer) [inline appengine/standard/mail/test/mocks/Message.php]: A lot of extra white space at the tops of classes/ functions, etc. Also, the indentation of the functions below is off (I assume a tabs issue?)

try [installing php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer#installation) and running this command:

```
php-cs-fixer fix --level=psr2 --fixers=concat_with_spaces,unused_use,trailing_spaces,indentation .
```

I'm not sure it'll fix *all* the issues, but it'll certainly help.

File: appengine/standard/mail/app.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+runtime: php55
+threadsafe: yes
+api_version: 1
+
+inbound_services:
+- mail
+- mail_bounce
+
+handlers:
```
Comment (bshaffer) [inline appengine/standard/mail/app.yaml]: we probably want to add the import tags for these to match our existing docs, i.e.

```yaml
# [START incoming_handler]
  - url: /_ah/mail/.+
    script: handle_incoming_email.php
# [END incoming_handler]
# [START bounce_handler]
  - url: /_ah/bounce
    script: handle_bounced_email.php
# [END bounce_handler]
```

**Comment by bshaffer** [comment]
closing in favor of #73

**Comment by tmatsuo** [comment]
Sorry guys we should skip the `gcloud auth activate-service-account --key-file /home/travis/build/GoogleCloudPlatform/php-docs-samples/credentials.json` call when it comes from other repo. A bug filed: #74

**Comment by bshaffer** [comment]
👍 thanks takashi

**Pull request closed without merging**